### PR TITLE
Cedar/Proto_PPP: Fix EAP identifier duplication

### PR DESCRIPTION
**Pull request by domosekai**
Currently EAP identifier starts from zero and causes duplication between the first packet (usually EAP Identity Request) and the second (usually EAP TLS Start).

This change uses unified identifier `NextId` which is already used by LCP and MSCHAPv2.

Moreover, EAP Success and Failure packets should use the same identifiers as Response so incorrect comments regarding hacks were removed.

Fixes https://github.com/SoftEtherVPN/SoftEtherVPN/issues/1350

﻿Changes proposed in this pull request:
 - Fix EAP identifier duplication and use unified identifier with LCP and MSCHAPv2

### CODE REVIEW

**Comment by davidebeatrici** [comment]
Thank you very much for your contribution!

File: src/Cedar/Proto_PPP.c [hunk 1]
```diff
@@ -3298,7 +3303,7 @@ bool PPPProcessEAPTlsResponse(PPP_SESSION *p, PPP_EAP *eap_packet, UINT eapTlsSi
 				else
 				{
 					PPP_PACKET *pack;
-					UINT identificator = p->Eap_PacketId - 1; // THIS IS A HACK TO SUPPORT VPN Client Pro on Android!!!
```
Comment (metalefty) [inline src/Cedar/Proto_PPP.c:3301]: Does this mean such a hack is no longer needed?

Comment (domosekai) [reply src/Cedar/Proto_PPP.c:3301]: This is not a hack actually. Unlike EAP Request where identifier should be increasing, EAP Success should use the same identifier as the previous Response. VPN Client Pro is behaving correctly.

Comment (domosekai) [reply src/Cedar/Proto_PPP.c:3301]: I removed this line because `Eap_PacketId` is not auto incremented now. `NextId` is.

**Comment by Evengard** [comment]
@davidebeatrici @metalefty Could you please ping me next time when something is wrong with Proto_PPP? I accidentally stumbled upon this pull request, but the information here is really important for me (also because I actually wrote the code, so I feel like it would be nice to be notified of all related bugs).

**Comment by davidebeatrici** [comment]
Sure!

**Pull request merged**